### PR TITLE
sql: add concurrent execution of `ADD INDEX` and `GENERAL` 

### PR DESCRIPTION
**Pull request by CaitinChen**
Via https://github.com/pingcap/docs-cn/pull/1021
@lilin90 @zimulala PTAL

### CODE REVIEW

File: sql/ddl.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ category: user guide
 
 DDL (Data Definition Language) is used to define the database structure or schema, and to manage the database and statements of various objects in the database.
 
+Currently, TiDB has implemented parallel execution of the `ADD INDEX` operation and a `GENERAL` operation (namely a non-`ADD INDEX` DDL operation) across different tables. In this case, two workers handle the `ADD INDEX` operation and a `GENERAL` operation respectively; when the operation requests are for the same table, workers execute these operations in the order of receiving the DDL operation requests. This feature is to guarantee the `ADD INDEX` operation (its execution time is longer that others' in TiDB) will not block other DDL operations.
```
Comment (morgo) [inline sql/ddl.md]: I think this should be called "concurrent execution".

Comment (gregwebs) [reply sql/ddl.md]: I agree that concurrent is more accurate.

Comment (shenli) [reply sql/ddl.md]: Yes

Comment (CaitinChen) [reply sql/ddl.md]: @morgo @gregwebs @shenli Thanks.

File: sql/ddl.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ category: user guide
 
 DDL (Data Definition Language) is used to define the database structure or schema, and to manage the database and statements of various objects in the database.
 
+Currently, TiDB has implemented concurrent execution of the `ADD INDEX` operation and a `GENERAL` operation (namely a non-`ADD INDEX` DDL operation) across different tables. In this case, two workers handle the `ADD INDEX` operation and a `GENERAL` operation respectively; when the operation requests are for the same table, workers execute these operations in the order of receiving the DDL operation requests. This feature is to guarantee the `ADD INDEX` operation (its execution time is longer that others' in TiDB) will not block other DDL operations.
```
Comment (lilin90) [inline sql/ddl.md]:
```suggestion
Currently, TiDB has implemented concurrent execution of the `ADD INDEX` operation and the `GENERAL` operation (namely the non-`ADD INDEX` DDL operation) across different tables. In this case, two workers process the `ADD INDEX` operation and a `GENERAL` operation respectively. When the operation requests are on the same table, workers execute these operations in the order of receiving the DDL operation requests. This feature is to guarantee the `ADD INDEX` operation (its execution time is longer than that of other operations in TiDB) does not block other DDL operations.
```

**Comment by CaitinChen** [comment]
@lilin90 PTAL

**Comment by CaitinChen** [comment]
@zimulala PTAL

**Pull request merged**